packages/date-time-picker/src/vaadin-date-time-picker.js

```diff
@@ -5,6 +5,7 @@
  */
 import './vaadin-date-time-picker-date-picker.js';
 import './vaadin-date-time-picker-time-picker.js';
+import { FlattenedNodesObserver } from '@polymer/polymer/lib/utils/flattened-nodes-observer';
```

Comment:
PLease add `.js` extension:

```suggestion
import { FlattenedNodesObserver } from '@polymer/polymer/lib/utils/flattened-nodes-observer.js';
```